app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+    @ingredients = @recipe.find_ingredients
```

Comment:
You don't necessarily need to create an instance variable for ingredients here because you can get to ingredients through a recipe (reaching through objects). So in your view you would just iterate over `@recipe.ingredients`